src/features/operator/Market/MarketOverview/index.tsx

```diff
@@ -79,6 +90,9 @@ export const MarketOverview = (): JSX.Element => {
           </Col>
           <Col span={12} style={{ textAlign: 'right' }}>
             <Space size={16}>
+              <Button icon={<InfoCircleOutlined />} onClick={showAssetInfoModal}>
```

Comment:
I would put the icon info without the button, and move it near the settings wheel